src/nerve/frontends/tui/commander/input_dispatcher.py

```
@@ -71,6 +86,39 @@ def _validate_and_create_error_block(
+        # Store lightweight metadata in block (no LLM data - keeps TUI fast)
+        block.metadata["suggestion"] = record.to_lightweight_dict()
+
+        # Fire-and-forget async append to JSONL
+        if is_history_enabled():
+            asyncio.create_task(
+                append_to_history_async(
+                    record,
+                    self.commander._suggestions.server_name,
+                    self.commander._suggestions.session_name,
+                )
+            )
```

Ruff (0.14.10)

Store a reference to the return value of `asyncio.create_task`

(RUF006)